src/redux/reducers.js

```diff
@@ -1233,3 +1361,18 @@ export function usernameSubscribers(state = {}, action) {
 export function usernameSubscriptions(state = {}, action) {
   return handleSubs(state, action, ActionTypes.SUBSCRIPTIONS)
 }
+
+export function compensateScroll(state = {}, action) {
+  switch(action.type){
+    case ActionTypes.SOCKET_POST_NEW: {
+      return {...state, postId: action.post.id}
+    }
+    case ActionTypes.SOCKET_COMMENT_NEW: {
```

Comment:
I think these two event names are the reason why scroll compensation doesn't work.